src/main/java/org/sonarjs/cli/analysis/SonarLint.java

```diff
@@ -48,13 +48,6 @@ public void runAnalysis(Map<String, String> properties, InputFileFinder finder,
       throw new IllegalStateException("Error preparing list of files to analyze", e);
     }
 
-    if (inputFiles.isEmpty()) {
-      LOGGER.warn("No files to analyze");
-      return;
```

Comment:
@inverno won't we miss logic of this `return`?

Reply:
@vilchik-elena nah, without the log part it became just a micro-optimization to not call the engine